src/filesystem/impls/filer/BlobUtils.js

```diff
@@ -68,10 +67,40 @@ define(function (require, exports, module) {
         if(!url) {
             return filename;
         }
-
         return url;
     }
 
+    function _getUrlAsync(filename, callback) {
+        var cachedUrl = blobURLs[filename];
+        if(cachedUrl !== filename) {
+            callback(null, cachedUrl);
+            return;
+        }
+
+        Filer.fs.readFile(filename, null, function(err, data) {
```

Comment:
This is still wrong.  I need to figure out a solution for getting `fs()` in here, which means fixing the circular ref between `BracketsFiler` and `Handlers`